anvil-python/anvil/commands/haproxy.py

```diff
@@ -42,12 +42,17 @@
 HAPROXY_UNIT_TIMEOUT = (
     1200  # 15 minutes, adding / removing units can take a long time
 )
-LOG = logging.getLogger(__name__)
+VALID_TLS_MODES = ["termination", "passthrough"]
 
 
-def validate_cert_file(filepath: str) -> None:
-    if filepath == "":
-        return
+def validate_cert_file(filepath: str | None) -> None:
+    if filepath is None:
```

Comment:
Why do we need to check on None rather than empty string? I am asking since `None` is bringing up the prompt when trying to apply a manifest that does not want to set ssl cert and path. Including the default one. Which is wrong behavior and that's why the change to `""` was chosen as part of #59

Reply:
I think I got confused while rebasing and accidentally reverted this back to checking `None`. I've changed it back now.